src/options.js

```diff
@@ -18,7 +18,7 @@ export var Options = {
 	followMarker: true,
 	imperial: false,
 	legend: true,
-	handlers: ["Distance", "Time", "Altitude", "Slope", "Speed", "Acceleration"],
+	handlers: ["Distance", "Time", "Altitude", "Slope", "Speed", "Acceleration", 'Temperature'],
```

Comment:
@xneek I need to double check the example, but I don't think I'll keep it as default (eg. like `pace` or `linear-gradient` handlers)